py/desispec/zcatalog.py

```diff
@@ -193,35 +196,49 @@ def create_summary_catalog(specprod, specgroup = 'zpix', \
     """
 
     ############################### Checking the inputs ##################################
-    
-    ## Initial check 1
-    ## Test whether the specprod exists or not
-    ## Spectral Directory Path for a given internal release name
-    specred_dir = specprod_root(specprod)
-    if (os.path.isdir(specred_dir) == False):
-        log.error(f'"{specprod}" directory does not exist')
-        raise ValueError(f'"{specprod}" directory does not exist')
 
-    ## Initial check 2
+    ## Initial check 1
     ## If specgroup = something else by mistake
     valid_specgroups = ('zpix', 'ztile')
     if specgroup not in valid_specgroups:
         errmsg = f'{specgroup=} not recognized, should be one of {valid_specgroups}'
         log.error(errmsg)
         raise ValueError(errmsg)
 
-    ######################################################################################
+    ## set indir if needed
+    if indir is None:
+        indir = specprod_root(specprod) + '/zcatalog'
```

Comment:
You know, if you're going to join paths this way, you might as well do it with `PathLib`.

Reply:
And you don't need to take the comment above particularly seriously; don't hold up the PR.

Reply:
It's admittedly a matter of taste/familiarity, but my complaint with pathlib.Path is that it overrode the division operator '/' to chain things together rather than '+' like strings have done since time immemorial.  My brain trips over whether the slash is incorrectly trying to divide strings, or whether it is a typo and was meant to be within a quotes like this case, or whether it actually is doing something fancy with overriding the / operator and relying upon left-to-right evaluation like `pathlib.Path('blat') / 'foo'`, which is especially confusing to me when it is `importlib_resources.files('blat') / 'foo'` where I have to independently know that `importlib_resources.files` is returning a `pathlib.Path` object with those properties.

i.e. I'm going to leave this as-is and declare it ok to assume that we will always be running DESI code on a posix-like filesystem with '/' as the directory hierarchy separator.